test/db/formats/pe/dotnet20

```diff
@@ -1,6 +1,12 @@
 NAME=PE: corkami dotnet20.exe - open
 FILE=bins/pe/dotnet20.exe
-CMDS=q!
+CMDS=<<EOF
+s sym.HelloWorld::Main; s
```

Comment:
I think, we need a bit more tests than this, especially in terms of the CIL bytecode itself, like `rz-asm` tests, also a bit more on the analysis